WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -24,21 +24,38 @@
 - [ ] Verify that you can delete a bookmark in edit mode
 - [ ] Verify that you can delete a bookmark folder with bookmarks inside
 - [ ] Verify adding a bookmark domain, subpaths is retained and you are successfully able to visit the domain subpath in a new tab
+- [ ] Verify adding bookmark for many open tabs works. Ensured opening all bookmarks in `Saved tabs` folder will load all the tabs
+- [ ] Verify import/export bookmarks works
 
 ## Favourites
 
 - [ ] Verify editing favourites and changing their URLs updates the favicons accordingly
 - [ ] Verify that you can remove favourites
 - [ ] Verify that you can add new favourites from the `Share with...` menu
+- [ ] Verify adding Favorites Widget to home screen and ensure favorites are updated on the widget accordingly
```

Comment:
Consistency
Favorites -> favorites

Please update the section to use `favorites` as that is the text in the app.